agent/util-scripts/pbench-tool-meister

```diff
@@ -361,9 +361,10 @@ class ToolMeister(object):
         else:
             return benchmark_run_dir, channel, controller, group, hostname, tools
 
-    def __init__(self, pbench_bin, params, redis_server, logger):
+    def __init__(self, pbench_bin, params, redis_server, tool_metadata, logger):
```

Comment:
if we can get tool_metadata from redis_server `redis_server.get("tool-metadata")` why do we need to pass it as a separate parameter?

Reply:
This will be changed in favor of a proper metadata class, currently just a relic of the old PR